integrator.py

```diff
@@ -214,15 +220,21 @@ def train_with_context(self, context: np.ndarray, batch_size=100, lr=None, point
 
             # --------------- START TODO compute loss ---------------
             y = batch_y.to(self.device)
-            mean = torch.mean(y / absdet)
-            var = torch.var(y / absdet)
-            y = (y / mean).detach()
+            y = y + np.finfo(np.float32).eps
+
+            log_y = torch.log(y)
```

Comment:
This increasing computation graph.

Reply:
At the moment, it doesn’t matter. We'll optimize after obtaining the correct results.